bigbluebutton/bigbluebutton
# Move the welcome message out of the chat area

### What does this PR do?

- implements session details modal
- removes welcome message from the chat area

![Screenshot from 2024-12-16 11-08-54](https://github.com/user-attachments/assets/4e4ae018-f41b-43a3-b661-eac0908c1d5d)


### Closes Issue(s)
Closes #21886

### How to test
1. create a meeting with the create parameters: 
```
moderatorOnlyMessage=This message should only appear for moderators
loginURL=https://test.com/test
dialNumber=555-555-1234
```

2. join the meeting
3. see the session details modal
